content/docs/1.0.x/reference/api/index.md

```diff
@@ -29,7 +29,10 @@ The Keptn API is documented in terms of a [Swagger API documentation](https://sw
     CLI is authenticated against the Keptn cluster http://YOUR.DOMAIN/api
     ```
 
-* Access the Keptn Swagger API documentation in your browser at: http://YOUR.DOMAIN/api
+* Access the Keptn Swagger UI and API documentation
+* by clicking on "Keptn API" in the top right menu on the Bridge
+* or by pointing your browser at: http://YOUR.DOMAIN/api .
```

Comment:
This way, it is indented and not at the same level and the last paragraph is not belonging to a bullet point as on the screenshot :) 
![image](https://github.com/keptn/keptn.github.io/assets/992621/c176967b-5cca-4ba9-973e-3f50131ca500)

```suggestion
    * by clicking on "Keptn API" in the top right menu on the Bridge
    * or by pointing your browser at: http://YOUR.DOMAIN/api .

```